Content.Server/GameObjects/EntitySystems/Click/InteractionSystem.cs

```diff
@@ -28,33 +28,33 @@ namespace Content.Server.GameObjects.EntitySystems
     /// <summary>
     /// This interface gives components behavior when being clicked on or "attacked" by a user with an object in their hand
```

Comment:
Docs need updating.

Reply:
actually it would be cool to write in comments which kind of SS13 procs these methods replicate

Reply:
@Injazz Probably better done in a different PR, since I don't know the SS13 equivalents